app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,40 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+  before_action :find_user, only: [:create]
```

Comment:
only 1 action thì k cần [ e
```suggestion
  before_action :find_user, only: :create
```